routers/web/repo/editor.go

```diff
@@ -190,6 +207,9 @@ func editFile(ctx *context.Context, isNewFile bool) {
 	ctx.Data["LineWrapExtensions"] = strings.Join(setting.Repository.Editor.LineWrapExtensions, ",")
 	ctx.Data["EditorconfigJson"] = GetEditorConfig(ctx, treePath)
 
+	ctx.Data["CanCreateNewBranch"] = !ctx.FormBool("hide_create_new_branch")
+	ctx.Data["CanCreatePullRequest"] = !ctx.FormBool("hide_create_pr")
```

Comment:
These variable names do not seem right.

`Can` usually means it is from permission checks. But here it is a UI toggle option.

Reply:
ec6ab8d2d5a8c2b01b7288fd1d7d58998e50de79